raiden/api/rest.py

```diff
@@ -84,8 +95,8 @@ def add_resource(self, resource_cls):
             resource_cls._route
```

Comment:
This is  how I received it from Max. It works and has the ability to load different api versions (when we have them) by just changing a few imports.

I don't know if there is a proper flask/flask-restful way to do it better or in a different way. What is bothering you with that here?

The documentation is good enough for some things, but for some others is lacking a lot and I had to step through their code to understand how their library works.

I don't want to waste more time fighting their library to make it do what I want it to do for our project if it works now. 

Ofcourse if you have a specific suggestion please go ahead.